### PR TITLE
Application Project Submission

### PR DESCRIPTION
**Pull request by jopet-tolentino**
What design patterns did you use inside the application?
Tabbed Activity with navigation style of Action Bar Tabs was the design to switch between Notes and Tasks.
ListFragments were used to show the list of notes and task each with their own listviews.
Custom Dialog and Alert Dialog were used to interact with users in order to create, update and delete.
The color palette used in the project can be seen at http://www.materialpalette.com/blue-grey/indigo.

What libraries did you use?
Why did you use this library?
Are there other libraries available?
If yes, what made you choose this library over the others?
Sqlite library was used as the database which stored the notes and tasks. 
This was chosen due to familiarity and can perform the specifications efficiently.
Plus Sqlite is already part of the Android environment. There's no need to add it in.
There are other ways to store data but Sqlite is the best fit for the app.

What major challenges did you encounter when making the app?
Other than the design, since I am more of a back-end developer. Dealing with fragments is always a challenge. :)

If you had more time, what additional features would you like to add?
I would like to add settings where the user can change the font, order displays, etc. of the list.
Also multiple item selection and a delete all function can be added.

The app consists of two major activities:
 a note-taking activity that accepts a title and text for one or more notes
 a TODO tracking activity that accepts an entry, with a checkbox to show whether this entry has been done or not done
The app has a note taking and TODO tracking activity. Users can add using the blue button with a plus sign at the right bottom of the screen.

The app should have a user interface that allows for easy access between the note-taking and TODO tracking activities as well as to see their details at a glance.
The user can swipe in between fragments to see all the details of the notes and TODOs at a glance.

The note-taking activity should provide a view for its notes in a reverse-chronological order, with options for updating and deleting old notes.
A list is provided to show all notes in a reverse chronological order (newest first). 
Clicking the items will show the user all the details of the notes and will have the option to delete and update the notes.

The TODO-tracking activity should provide a view for its entries with options for updating and deleting old TODOs.
A list is provided to show all TODOs in chronological order (oldest first). 
Clicking the items will show the user all the details of the TODOs and will have the option to delete and update the TODOs.

The app should make it easy to switch between the note-taking and TODO-tracking activities.
Switching between the activities can be done by clicking the tabs at the top or by swiping.

The app should make it easy to switch a note to a TODO entry, and vice versa.
The user can to switch a note to a TODO entry, and vice versa by doing a long press on an item on the list.

The release APK of the app should take up no more than 3MB of disk space.
The release APK is about 1.6MB of disk space.

### CODE REVIEW

File: local.properties [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## This file is automatically generated by Android Studio.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file should *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+sdk.dir=C\:\\Users\\Jopet\\AppData\\Local\\Android\\sdk
```
Comment (zakame) [inline local.properties:10]: This should not be committed in.

File: gradlew [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+fi
+
+# Split up the JVM_OPTS And GRADLE_OPTS values into an array, following the shell quoting and substitution rules
+function splitJvmOpts() {
+    JVM_OPTS=("$@")
+}
+eval splitJvmOpts $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS
+JVM_OPTS[${#JVM_OPTS[*]}]="-Dorg.gradle.appname=$APP_BASE_NAME"
+
+exec "$JAVACMD" "${JVM_OPTS[@]}" -classpath "$CLASSPATH" org.gradle.wrapper.GradleWrapperMain "$@"
```
Comment (zakame) [inline gradlew:160]: This file should be set executable on OS X / Linux.

**Comment by zakame** [comment]
the built APK file should not be included in this PR.

File: app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
+
+    defaultConfig {
+        applicationId "com.yoyo.jopet.androidorgapp"
+        minSdkVersion 21
```
Comment (zakame) [inline app/build.gradle:9]: Minimum version is rather new (Lollipop and above) which means no love for KitKat and below... :crying_cat_face:

**Comment by jopet-tolentino** [comment]
Will take note of these comments for future uploads (first time using github as version control). Thanks. 

I can change the minSdkVersion, I don't think it will affect much if I lower it down. My test device is lollipop so I didn't bother.

Have a nice day!

**Pull request closed without merging**